docs: add change log cn for v4.3.15 and e4.3.10 (#1165)
emqtt/emq-docs-cn: en_US/changes/changes-4.3.md

# Changes

## Version 4.3.15

*Release Date: 2022-06-01*

### Enhancement

> - Add more time transformation functions to the SQL of rule engine

Reviewer: maybe give an example date transform function.
also hint `data` command formatter string support ?

- Add the `float2str/2` function to the SQL of rule engine to support specifying the output precision of floating point numbers
- Support for using JWT for authorization, now MQTT clients can authorize using specific claims that include a pub-sub whitelist
- Improved authentication related metrics to make it easier to understand, now `client.authenticate = client.auth.success + client.auth.failure`

> - Support binding the listener of the REST API to the specified network interface

Reviewer: ```suggestion
- Support binding the listener of the REST API to a specific network interface
```

- Support multi-condition query and fuzzy query for user data in authentication and authorization using built-in database as data source
- Supports querying clients using the length of the message queue and the number of dropped messages as conditions
- Support to configure the log time format to be compatible with the time format in older versions
- When `use_username_as_clientid` is configured to `true` and the client connects without specifying a `username`, the connection is now rejected with a reason code `0x85`

> - Improve random performance of AppSecret

Reviewer: ```suggestion
- Full randomisation of app secrets (previously partially randomised)
```

- Hot upgrades between incompatible versions will now be rejected

> - Now EMQX's startup path will allow to contain spaces

Reviewer: ```suggestion
- Allow white spaces in EMQX's installation path
```

> - Add check for node name

Reviewer: ```suggestion
- Boot script fail fast on invalid node name (improve error message readability)
```

### Bug fixes

> - Fix the issue that rule engine's SQL function `hexstr_to_bin/1` could not handle half byte

Reviewer: ```suggestion
- Fix the issue that rule engine's SQL function `hexstr_to_bin/1` could not handle half-byte.
```
